### ISSUE TITLE
You need more layers or elements to grow your edition to 9984 artworks!

### ISSUE THREAD

**Issue by samdupnamgyal** [HashLips/hashlips_art_engine#1108]
i have 5 layers of 
4x back ground colours
4x faces
4x series
1x border
156 numbers
4x4x4x1x156 = 9984 
so why am i not able to grow it enough?

**Comment by bolshoytoster** [comment]
@samdupnamgyal it's because of how hashlips generated dna. You can disable this by removing the `failedCount++;` ~line 418 `sr/main.js`:
https://github.com/HashLips/hashlips_art_engine/blob/d8ee279043d2d4a8de3bdfac0d89d0e966fb04a2/src/main.js#L417-L419

**Comment by samdupnamgyal** [comment]
thank you so much it worked just hit my target of 9984!

**Comment by steveDL** [comment]
> 

Hey Does this not mean it would be able to create duplicates though?

**Comment by bolshoytoster** [comment]
@steveDL no, it won't create duplicates, but that means it would take quite a while to create a large collection.
